### ISSUE TITLE
Laravel 7.3 - ramsey/uuid issue

### ISSUE THREAD

**Issue by quaidesbalises** [archtechx/tenancy#338]
Hi,

There's an issue when installing the package on the new 7.3 due to ramsey/uuid.

```
 Problem 1
    - Conclusion: don't install stancl/tenancy v2.3.0
    - Conclusion: remove ramsey/uuid 4.0.0
    - Installation request for stancl/tenancy ^2.3 -> satisfiable by stancl/tenancy[2.x-dev, v2.3.0].
    - Conclusion: don't install ramsey/uuid 4.0.0
    - stancl/tenancy 2.x-dev requires ramsey/uuid ^3.7 -> satisfiable by ramsey/uuid[3.7.0, 3.7.1, 3.7.2, 3.7.3, 3.8.0, 3.9.0, 3.9.1, 3.9.2, 3.9.3, 3.x-dev].
    - Can only install one of: ramsey/uuid[3.7.0, 4.0.0].
    - Can only install one of: ramsey/uuid[3.7.1, 4.0.0].
    - Can only install one of: ramsey/uuid[3.7.2, 4.0.0].
    - Can only install one of: ramsey/uuid[3.7.3, 4.0.0].
    - Can only install one of: ramsey/uuid[3.8.0, 4.0.0].
    - Can only install one of: ramsey/uuid[3.9.0, 4.0.0].
    - Can only install one of: ramsey/uuid[3.9.1, 4.0.0].
    - Can only install one of: ramsey/uuid[3.9.2, 4.0.0].
    - Can only install one of: ramsey/uuid[3.9.3, 4.0.0].
    - Can only install one of: ramsey/uuid[3.x-dev, 4.0.0].
    - Installation request for ramsey/uuid (locked at 4.0.0) -> satisfiable by ramsey/uuid[4.0.0].
```

**Comment by stancl** [comment]
Can you try deleting `composer.lock`, `vendor/`, adding `stancl/tenancy` to `composer.json` manually and running `composer install`?

Seems like your composer downloaded v4 before installing stancl/tenancy which requires v3.

I'll update my package to work with both versions.

**Comment by stancl** [comment]
Will be fixed in 2.3.1.
